build.ts

```diff
@@ -28,32 +29,19 @@ interface GitHubRelease {
   "assets": GitHubAsset[];
 }
 
-const gitHubAPIFetchOptions = {
-  headers: {
-    accept: "application/vnd.github.v3+json",
-  },
+const gitHubAPIHeaders = {
+  accept: "application/vnd.github.v3+json",
 } as const;
 
-console.log(`${colors.bold(colors.green("Building"))} deploy_doc...`);
+$.logStep("Building deploy_doc...");
```

Comment:
Still works (tested this new build script on an old commit of this repo):

![image](https://user-images.githubusercontent.com/1609021/179579215-5dce9c3c-bf77-4a95-afb1-ce9094cd569f.png)